app/controllers/triannon/auth_controller.rb

```diff
@@ -5,64 +5,56 @@ module Triannon
   class AuthController < ApplicationController
     include RdfResponseFormats
 
-    AUTH_EXPIRY  =   60 # seconds
-    TOKEN_EXPIRY = 3600 # seconds
+    # HTTP request methods accepted by /auth/login
+    LOGIN_ACCEPT = 'OPTIONS, GET, POST'
 
     # OPTIONS /auth/login
     def options
```

Comment:
Right, the IIIF spec is not specific about the HTTP method to be used, but it does specify the data payload (info.json) required to describe the service, e.g. http://image-auth.iiif.io/api/image/2.1/authentication.html#login-service;  so this implementation can return the description (info.json) in response to an OPTIONS request.  @azaroth42 might care to comment on this and/or advise on what HTTP method(s) should return info.json data.